WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController+QuickActions.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import WordPressShared
+
+// MARK: - Quick Actions
+
+extension BlogDashboardViewController {
+
+    func showStats() {
```

Comment:
Do you think we can move that to the `DashboardQuickActionsCardCell`?

The cell has the `Blog` and the `ViewController` injected in it, I feel we can have this logic there.

I like this approach because:

* It decouples the cell from `BlogDashboardViewController`
* It makes this cell reusable anywhere else, as long as a blog and a viewController is provided
* We avoid similar issues we have a few days ago with `BlogDetailsViewController` (in which a lot of logic was tied to this VC)